prijateli_tree/app/templates/round.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+{% block title %}
+  PrijateliTree - View Round
+{% endblock title %}
+{% block content %}
+  <div style="text-align: center;">
+    <h1 style="margin-bottom: 30px;">Round: {{ current_round }}</h1>
+    {% if first_round %}
+      <p style="font-size: 24px; margin-bottom: 20px;">
+        Your Ball:
+        {% if ball == 'R' %}
+          <span style="color: red; font-size: 24px;">Red</span>
+        {% elif ball == 'B' %}
+          <span style="color: #007bff; font-size: 24px;">Blue</span>
+        {% else %}
+          {{ ball }}
+        {% endif %}
+      </p>
+    {% else %}
+      <h2 style="margin-bottom: 20px;">Your Previous Guess</h2>
+      <p style="font-size: 24px; margin-bottom: 20px;">
+        {% if previous_answers.your_previous_answer == 'R' %}
+          <span style="color: red;">Red</span>
+        {% elif previous_answers.your_previous_answer == 'B' %}
+          <span style="color: #007bff;">Blue</span>
+        {% else %}
+          {{ previous_answers.your_previous_answer }}
+        {% endif %}
+      </p>
+      <h2 style="margin-bottom: 20px;">Neighbors' Previous Guesses</h2>
+      <ul style="list-style: none; padding: 0;">
+        {% for neighbor_answer in previous_answers.neighbors_previous_answer %}
+          <li style="font-size: 24px; margin-bottom: 20px;">
+            {% if neighbor_answer == 'R' %}
+              <span style="color: red;">Red</span>
+            {% elif neighbor_answer == 'B' %}
+              <span style="color: #007bff;">Blue</span>
+            {% else %}
+              {{ neighbor_answer }}
+            {% endif %}
+          </li>
+        {% endfor %}
+      </ul>
+    {% endif %}
+  </div>
```

Comment:
There should be a choice at the bottom as well -- so two buttons or for now just links: red or blue. 
See the UX slide 36: https://drive.google.com/file/d/19CdIVZfstrk1I3NiyX2mICZzNgc-w0ww/view

This can be a new issue.

Reply:
Added dummy buttons for now